### ISSUE TITLE
No such file or directory (yaml file)

### ISSUE THREAD

**Issue by boryszef** [nokia-wroclaw/innovativeproject-django-sourcerer#9]
On fresh installation of django-sourcerer 1.1 in py project, I've got the following error:
```shell
  File "/home/borys/github/django-sourcerer-test/venv/lib64/python3.6/site-packages/django_sourcerer/import_datas/parsing_config_file.py", line 10, in import_yaml_file
    with open(self.parse, 'r') as stream:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/cemarslan/Desktop/innovativeproject-django-sourcerer/django_sourcerer/csv_config_file.yaml'
```

**Comment by cmarslan** [comment]
Issue fixed at 1.3 version of project

**Comment by cmarslan** [comment]
fixed issue
